src/ol_infrastructure/applications/concourse/__main__.py

```diff
@@ -249,6 +246,17 @@
     vpc_id=ops_vpc_id,
 )
 
+ec2.SecurityGroupRule(
+    "concourse-worker-access-from-concourse-web",
+    security_group_id=concourse_worker_security_group.id,
+    source_security_group_id=concourse_web_security_group.id,
+    protocol="tcp",
+    from_port=0,
+    to_port=65535,
```

Comment:
Does access have to be that open?

Reply:
Most likely not, I'll test with putting it back to using port 2222, but when I was working through getting things working it seemed like there might have been some additional port issues. That being said, it still only allows traffic between the web and worker instances and not any other sources.